### PR TITLE
Quiz submission

### PR DESCRIPTION
**Pull request by victorhadomskyi**

### CODE REVIEW

File: TestingTool/TestingTool/Models/Answer.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@ public partial class Answer
     {
         public int AnswerId { get; set; }
         public string Text { get; set; }
+        // #TODO: Should it be nullable?
```
Comment (IvanovVadym) [inline TestingTool/TestingTool/Models/Answer.cs]: not sure
let's leave it as `bool`

File: TestingTool/TestingTool/Dtos/QuizQuestionSubmissionDto.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace TestingTool.Dtos
+{
+    public sealed class QuizQuestionSubmissionDto
```
Comment (IvanovVadym) [inline TestingTool/TestingTool/Dtos/QuizQuestionSubmissionDto.cs]: Do we need this DTO if it's the sane as `QuizQuestionSubmission`

File: TestingTool/TestingTool/Dtos/QuizSubmissionDto.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace TestingTool.Dtos
+{
+    public sealed class QuizSubmissionDto
```
Comment (IvanovVadym) [inline TestingTool/TestingTool/Dtos/QuizSubmissionDto.cs]: the same for `QuizSubmission`
Probably I miss something 😢

Comment (victorhadomskyi) [reply TestingTool/TestingTool/Dtos/QuizSubmissionDto.cs]: I wanted to be consistent with all the rest project structure.
Also there can be possible modifications in these files.
I'll better add a comment desribing these things

Comment (victorhadomskyi) [reply TestingTool/TestingTool/Dtos/QuizSubmissionDto.cs]: In such a case we can simply remove one of them later on :)

**Pull request merged**